### ISSUE TITLE
[Organizer Request] Add or Remove Mateo Lopez Pozo to Cuenca chapter

### ISSUE THREAD

**Issue by MateoLPozo2** [cncf/communitygroups#245]
### Name of chapter inquiring about

Cloud Native Cuenca

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Mateo Lopez Pozo, https://github.com/MateoLPozo2, www.linkedin.com/in/mateo-lopez-pozo, Mateo Lopez Pozo in CNCF Slack, https://community.cncf.io/u/mjfm7y/#/about

### Tell us why you want to be added or removed from the chapter

I, Mateo López Pozo, am eager to join as an organizer to engage in meaningful projects, contribute as a source of knowledge for my country, and gain valuable experience in organizational management.

I bring experience in managing teams through volunteer projects and possess broad technical knowledge. I am currently pursuing two bachelor’s degrees in Information Technology and Computer Science in Ecuador and Germany, showcasing my dedication and commitment to the field.

I believe that CNCF presents an excellent opportunity to raise awareness of cloud computing and expand its accessibility to wider audiences, driving innovation and adoption in this crucial area.

### Code of Conduct

- [x] I agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizer resides

- [ ] I confirm I am local to or live near the city of the above-proposed chapter (do not mark if not relevent to request)

### Organizers companies/affiliations

- [x] I come from xyz company which is differing from that of the current organizers (do not mark if not relevent to request)

### Inclusive

- [x] I agree. To assure this, I have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) (do not mark if not relevent to request)

### Vendor Neutral

- [x] I have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and I agree the main content of our meetups will remain vendor-nuetral (do not mark if not relevent to request)

### Remain an active chapter

- [x] I understand the activity level standards of all CNCF chapters (do not mark if not relevent to request)

### Collaboration with other chapters

- [x] I acknowledge the CNCF right to approve additional chapters in my area based on this criteria (do not mark if not relevent to request)

### Aim to have a diverse representation of speakers in your meetups

- [x] I will do my best to host diverse speakers throughout the year (do not mark if not relevent to request)

**Comment by AudMonte01** [comment]
Hi @MateoLPozo2 - thank you for your request, but there are some check marks missing above. Do you have any questions about those?

Please let me know once you have completed and agreed to the above check marks and you can re-open this GitHub issue.

it is also helpful to tag me when you do re-open it. As well as get the [current organizers](https://community.cncf.io/cloud-native-cuenca/) to approve you by commenting on this issue as well.

**Comment by MateoLPozo2** [comment]
Hi @AudMonte01 - the remaining points have now been cross-checked. You will find the badge confirming completion of the required course attached. [LFC102](https://www.credly.com/users/mateo-lopez-pozo/badges#credly)
The listed resources have also been reviewed.

One thing I’d like to confirm is my current residence. I don’t currently live near the chapter’s city or locations, so my support would mainly be online. 

You can also confirm this with the current organizers. _I’ll reach out to them directly as well, since their profiles don’t list their GitHub to mention them directly._

Just wanted to raise this so we’re aligned.

**Comment by AudMonte01** [comment]
That is excellent news, @MateoLPozo2 !

I am copying the organizers here so they review your request and tell you next steps:
- @xllauca, can you also copy Anthony Grijalva since I don't have his GitHub handle? Thanks in advance.

**Comment by xavier-tipti** [comment]
Copying @AnthonyGrillo here. Thanks @AudMonte01!.

**Comment by xllauca** [comment]
I agree with adding Mateo Lopez Pozo (@MateoLPozo2) as an organizer for the Cuenca chapter

**Comment by AnthonyGrillo** [comment]
I agree with adding **Mateo Lopez @MateoLPozo2** as an organizer for the Cuenca chapter.

**Comment by AudMonte01** [comment]
Thank you, @AnthonyGrillo and @xllauca ! I have moved forward with adding @MateoLPozo2 as an chapter organizer. Mateo, you should have just received an automated welcome email from us with next steps. And if you agree to the above organizer terms, please check the box upon accessing the chapter via Bevy.

Also, can you link me to your CNCF Slack Workspace profile? I cannot find it with a name search.

**Comment by xllauca** [comment]
Thank you so much! It's great to have Mateo on board.
Regarding Slack, I had some trouble joining the workspace previously, which is likely why I didn't show up in the search. Here are my details:

Slack ID: U050RA0CW66
Profile Link: https://cloud-native.slack.com/team/U050RA0CW66

Best regards!

**Comment by AudMonte01** [comment]
Thank you, @xavier-tipti for also sending your Slack profile. Will you please add a unique image so it is easier to add you? Believe it or not, there are many Xavi's without a profile picture in the CNCF Slack workspace. Thanks in advance!

@MateoLPozo2 - please send along a link to your Slack profile as well, or let me know if we should send you an invite to the workspace.

**Comment by MateoLPozo2** [comment]
Hi @AudMonte01 just created an Slack account, read the terms and conditions for organizers, accepted those, and got access to the dashboard, and 
- here is the identifier:U0AQ01Z5M2T
- and profile link: https://cloud-native.slack.com/team/U0AQ01Z5M2T
Let me know if you need anything else from my side.

**Comment by AudMonte01** [comment]
Thank you! I have added both you and @xllauca to the organizer Slack channel. Hopefully I've added the correct account for Xavi.

@AnthonyGrillo - it would be great for you to send me your Slack profile as well.

Thanks in advance!
~Audra

**Comment by AnthonyGrillo** [comment]
Hi @AudMonte01  , thanks for the update!

I’ve joined the Slack workspace. Sharing my Slack contact details below:

Name: AnthonyGrijalva
Email: [smithanthony10@hotmail.com]



Please let me know if you need anything else from my side.

**Comment by AudMonte01** [comment]
Thank you, @AnthonyGrillo - I just invited you to the CNCF Slack Workspace. Here is an invite link in case it did not send you an email: https://join.slack.com/t/cloud-native/shared_invite/zt-3pyxvi8g5-rBbJc5ZFzMYjiM0QyGjLvA
